clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -114,8 +114,10 @@ public synchronized void add(String topic) {
      * is now
      */
     public synchronized long timeToNextUpdate(long nowMs) {
+        // Could be > 0 when update isn't requested explicitly and metadataExpiresMs isn't elapsed since the last successful update.
```

Comment:
Well, when I first read this method it was bit hard to understand which value would be adopted, so I thought small comments would help new readers but I'm fine to remove these comments if you think it's not that helpful.